Android/app/src/main/java/SampleMOdel.java

```diff
@@ -0,0 +1,53 @@
+import com.google.android.gms.maps.model.LatLng;
```

Comment:
I was trying parceler issue with the sample class... Just a trial class

Thanks,
Rakhee

On Mar 19, 2016, at 2:49 PM, Deepak Barge <notifications@github.com<mailto:notifications@github.com>> wrote:


In Android/app/src/main/java/SampleMOdel.java<https://github.com/deborshisaha/ModaRenta/pull/57#discussion_r56755647>:

> @@ -0,0 +1,53 @@
> +import com.google.android.gms.maps.model.LatLng;


What is the SampleModel needed for ? It just has LatLng ? Why do we need it
@rakheegupta<https://github.com/rakheegupta>

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/deborshisaha/ModaRenta/pull/57/files/b5a0e76ed0e25c34f4b9737bf76df95ed8676ffb#r56755647>